web/angular-wallet/src/app/network/network.service.ts

```diff
@@ -13,13 +13,13 @@ import {
 
 import {ApiService} from '../api.service';
 import { StoreService } from 'app/store/store.service';
-import { Settings } from 'app/settings';
+import { BehaviorSubject } from 'rxjs';
 
 
 @Injectable()
 export class NetworkService {
   private api: Api;
-  blocks: Block[];
+  public blocks: BehaviorSubject<any> = new BehaviorSubject([]);
 
   constructor(apiService: ApiService, private storeService: StoreService) {
```

Comment:
Now we expose a simple `BehaviorSubject` if anyone needs access to the most recent 100+ blocks.